vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher_prefix_list_updater.h

```diff
@@ -38,9 +38,6 @@ class PublisherPrefixListUpdater {
   // Starts the auto updater
   void StartAutoUpdate(PublisherPrefixListUpdatedCallback callback);
 
-  // Cancels the auto updater
-  void StopAutoUpdate();
```

Comment:
Even though you aren't using this anymore, I would consider leaving this method: the `PublisherPrefixListUpdater` abstraction is designed to allowing starting/stopping, which may be useful in the future.

Reply:
sure, added back